roles/convert2rhel/tasks/activation_keys.yml

```diff
@@ -31,8 +31,7 @@
     organization: "{{ foreman_organization }}"
     name: "{{ foreman_convert2rhel_key_centos7 }}"
     lifecycle_environment: "{{ foreman_convert2rhel_lifecycle_env }}"
-    content_view: "{{ foreman_convert2rhel_content_view }}"
```

Comment:
This CV is still used in the `host_groups` playbook, is that correct?!

Reply:
Yes it's correct. Now if I'm thinking about that, we should probably also change the CVs for the host groups, kinda forgot about it when was doing this PR